src/main/java/io/cdap/plugin/python/transform/JythonPythonExecutor.java

```diff
@@ -58,7 +60,17 @@ public void initialize(ScriptContext scriptContext) throws IOException, Interrup
     String script = String.format("%s\ntransform(%s, %s, %s)",
                                   config.getScript(), INPUT_STRUCTURED_RECORD_VARIABLE_NAME,
                                   EMITTER_VARIABLE_NAME, CONTEXT_NAME);
-    compiledScript = interpreter.compile(script);
+    try {
+      compiledScript = interpreter.compile(script);
+    } catch (PySyntaxError e) {
+      if (Strings.isNullOrEmpty(e.getMessage())) {
+        throw new RuntimeException(
+          "Unknown syntax error has occurred while interpreting python code. Please make sure syntax is correct.", e);
+      }
+      throw new RuntimeException(
+        String.format("Syntax error '%s' occurred while interpreting python code. " +
```

Comment:
if the error message is long, this will look funny. I think it's better if the message is at the end, something like:

'Syntax error occurred while interpreting python code: %s'